lib/linters/jslint/index.js

```diff
@@ -0,0 +1,43 @@
+// Load modules
+
+var Fs = require('fs');
+var Path = require('path');
+var Nodejslint = require('jslint');
+
+
+// Declare internals
+
+var internals = {};
+
+var formatErrors = function (error) {
+    return {
+        line: error.line,
+        severity: 'ERROR',
+        message: error.reason
+    };
+};
+
+var formatFile = function (file) {
+  return {
```

Comment:
Our styles a little different, 4 spaces and a newline after each function declaration.